### PR TITLE
Feat: img, card, input and text utilities added

### PR DESCRIPTION
**Pull request by Ashutosh-Bhadauriya**
Added image, card, input and textUtilities component

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+.img-dimensions {
+  width: 100%;
+  height: 12rem;
+}
+.hidden {
+  display: none;
+}
+.span.hidden {
+  display: none;
+}
```
Comment (OmKakatkar) [inline components/card/card.css:75]: The .span.hidden can be removed, as .hidden is doing the same work

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+.container {
+  width: 100%;
+  margin: 2rem auto;
+  padding: 2rem;
+  border-radius: 6px;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (OmKakatkar) [inline components/card/card.css:6]: This box-shadow seems to repeated on other pages as well, try to add a variable for the same.

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+.container {
+  width: 100%;
+  margin: 2rem auto;
+  padding: 2rem;  
+  border-radius: 6px;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (OmKakatkar) [inline components/input/input.css:6]: Reused box-shadow, can try adding variable for the same

File: components/textUtilities/textUtilities.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.container {
+  width: 100%;
+  margin: 2rem auto;
+  padding: 2rem;
+  border-radius: 6px;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (OmKakatkar) [inline components/textUtilities/textUtilities.css:6]: Repeated box-shadow

File: components/image/image.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.container {
+    width: 100%;
+    margin: 2rem auto;
+    padding: 2rem;  
+    border-radius: 6px;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (OmKakatkar) [inline components/image/image.css:6]: Repeated box-shadow

**Review by OmKakatkar — APPROVED** [review]
Just look into the suggestions. Otherwise, good to merge!

**Review by codewithrishu — APPROVED** [review]
always use alt in img tag. other then that looks good to merge

**Pull request merged**